.changelog/1329.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
this is not a bug; it's a new feature so `enhancement`.

Reply:
I updated it to enhancement.